data/json/items/gun/ups.json

```diff
@@ -99,7 +99,16 @@
     "ups_charges": 180,
     "reload": 0,
     "ammo_effects": [ "LASER", "INCENDIARY" ],
-    "flags": [ "NO_UNLOAD", "NEVER_JAMS", "MECH_WEAPON", "NO_UNWIELD", "NO_SALVAGE", "NO_REPAIR", "UNBREAKABLE_MELEE" ]
+    "flags": [
+      "NO_UNLOAD",
+      "NEVER_JAMS",
+      "MECH_WEAPON",
+      "NO_UNWIELD",
+      "NO_SALVAGE",
+      "NO_REPAIR",
+      "UNBREAKABLE_MELEE",
+      "NON-FOULING"
```

Comment:
I forgot this gun in my bugfix for fouling? 😱

Reply:
I think it just got re-added during merging.

Reply:
huh? all items in that file are flagged non-fouling. :confused:

Reply:
Referring to my not adding NON-FOULING to this gun in #35335